alfalfa_worker/worker_fmu/step_fmu.py

```diff
@@ -169,12 +169,12 @@ def run(self):
                         self.set_idle_state()
                         break
         else:
-            # first step - AP not sure this is necessary
-            # if self.simtime == self.startTime:
-            #     print("taking first step at {}",datetime.now())
-            #     self.step()
-            #     print("finished first step at {}",datetime.now())
-            #     next
+            # first step takes extra time
```

Comment:
Is this due to warmup in spawn?

Reply:
I'm not entirely sure.  it seems to happen after the Spawn warmup completes but it was taking a several minutes when i logged it.